src/Init/Init_FFTW.cpp

```diff
@@ -26,8 +26,19 @@ gramfe_fftw::complex_plan_1d FFTW_Plan_ExtPsi, FFTW_Plan_ExtPsi_Inv;   // ExtPsi
 //
 // Return      :  length of array that is large enough to store FFT input and output
 //-------------------------------------------------------------------------------------------------------
+//int ComputePaddedTotalSize(int* size) {
+//   return 2*(size[0]/2+1)*size[1]*size[2];
+//} // FUNCTION : ComputePaddedTotalSize
 size_t ComputePaddedTotalSize(int* size) {
```

Comment:
Since when using `MPI` the returned value here is the padded size required by each rank instead of the padded total size, so maybe we can rename it to something like `ComputePaddedRequiredSize` to avoid misunderstanding~?